### ISSUE TITLE
Button in settings window not visible under Ubuntu 15.10

### ISSUE THREAD

**Issue by nuest** [dlech/KeeAgent#143]
I just installed KeeAgent and noticed that the "Refresh" button is not visible in the window listing available keys (resizing the window did not help):

![keeagent-button-bug](https://cloud.githubusercontent.com/assets/1325054/13778404/db41513c-eab5-11e5-8af7-73f3f7ea5b20.jpg)

Versions:

```
...:~$ mono --version
Mono JIT compiler version 3.2.8 (Debian 3.2.8+dfsg-4ubuntu4)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           __thread
	SIGSEGV:       altstack
	Notifications: epoll
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	LLVM:          supported, not enabled.
	GC:            sgen
...:~$ lsb_release -a
LSB Version:	core-2.0-amd64:core-2.0-noarch:core-3.0-amd64:core-3.0-noarch:core-3.1-amd64:core-3.1-noarch:core-3.2-amd64:core-3.2-noarch:core-4.0-amd64:core-4.0-noarch:core-4.1-amd64:core-4.1-noarch:security-4.0-amd64:security-4.0-noarch:security-4.1-amd64:security-4.1-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 15.10
Release:	15.10
Codename:	wily
...:~$ keepass2 --version
Could not set X locale modifiers
KeePass 2.31
Copyright © 2003-2016 Dominik Reichl
```

**Comment by nifr** [comment]
I experienced the same issue on Linux and OSX. The hidden button is the **Refresh** button.

![screen shot 2016-03-19 at 13 19 40](https://cloud.githubusercontent.com/assets/851258/13897056/49fd569e-edd5-11e5-899f-4758bf8140dc.png)

Funnywise moving the mouse in circles over the button makes it appear sometimes. Then still only the little part of it on the left (that was already visible before) is actually clickable :grey_question: 

![screen shot 2016-03-19 at 13 20 17](https://cloud.githubusercontent.com/assets/851258/13897058/5f878d68-edd5-11e5-967b-2ee978bca5cf.png)

**Comment by dlech** [comment]
Fix has been released in KeeAgent v0.7.12. Please [download](http://lechnology.com/software/keeagent/#download) and try it out. Should also be available via ppa soon.
